po/pl.po

```diff
@@ -21,363 +21,363 @@ msgstr ""
 msgid "Inflow (%s)"
-msgstr ""
+msgstr "Przychód (%s)"
 
 #: src/view/TransactionView.vala:603
 msgid "Memo"
-msgstr ""
+msgstr "Notatka"
 
 #. right-click menu
 #: src/view/TransactionView.vala:618
 msgid "Split"
-msgstr ""
+msgstr "Podziel"
 
 #. right-click menu
 #: src/view/TransactionView.vala:620 src/view/Sidebar.vala:236
 msgid "Remove"
-msgstr ""
+msgstr "Usuń"
 
 #: src/view/TransactionView.vala:951
 msgid "Transaction removed"
-msgstr ""
+msgstr "Transakcja usunięta"
 
 #: src/view/Sidebar.vala:278
 #, c-format
 msgid "Budget overview for %s"
-msgstr ""
+msgstr "Przegląd budżetu dla %s"
 
 #: src/view/Sidebar.vala:282
 msgid "Spending this month"
-msgstr ""
+msgstr "Wydano w tym miesiącu"
 
 #: src/view/Sidebar.vala:291
 #, c-format
 msgid "Money spent in %s"
-msgstr ""
+msgstr "Pieniądze wydane w %s"
 
 #: src/view/Sidebar.vala:294
 msgid "Income this month"
-msgstr ""
+msgstr "Przychód w tym miesiącu"
 
 #: src/view/Sidebar.vala:303
 #, c-format
 msgid "Money earned in %s"
-msgstr ""
+msgstr "Pieniądze zarobione w %s"
 
 #. remaining
 #: src/view/Sidebar.vala:306 src/view/CategoryProperties.vala:125
 msgid "Remaining"
-msgstr ""
+msgstr "Pozostało"
 
 #: src/view/Sidebar.vala:315
 #, c-format
 msgid "Remaining balance for %s"
-msgstr ""
+msgstr "Pozostały bilans dla %s"
 
 #. Add "Accounts" category header
 #: src/view/Sidebar.vala:319
 msgid "Accounts"
-msgstr ""
+msgstr "Konta"
 
 #. Add "Add account..."
 #: src/view/Sidebar.vala:344
 msgid "Add account…"
-msgstr ""
+msgstr "Dodaj konto…"
 
 #: src/view/Sidebar.vala:353
 msgid "Add a new account"
-msgstr ""
+msgstr "Dodaj nowe konto"
 
 #. Add "Categories" category header
 #: src/view/Sidebar.vala:357
 msgid "Spending categories"
-msgstr ""
+msgstr "Kategorie wydatków"
 
 #. Add "Uncategorized"
 #: src/view/Sidebar.vala:464 src/window/MainWindow.vala:285
 msgid "Uncategorized"
-msgstr ""
+msgstr "Nieskategoryzowane"
 
 #. Add category...
 #: src/view/Sidebar.vala:493
 msgid "Add category…"
-msgstr ""
+msgstr "Dodaj kategorię…"
 
 #: src/view/Sidebar.vala:494
 msgid "Add a new spending category"
-msgstr ""
+msgstr "Dodaj nową kategorię wydatkową"
 
 #: src/view/Sidebar.vala:749
 msgid "None yet"
-msgstr ""
+msgstr "Jeszcze nic nie ma"
 
 #: src/view/Sidebar.vala:1127
 #, c-format
 msgid "Category %s removed"
-msgstr ""
+msgstr "Kategoria %s usunięta"
 
 #: src/view/Sidebar.vala:1138
 #, c-format
 msgid "Account %s removed"
-msgstr ""
+msgstr "Konto %s usunięte"
 
 #: src/view/WelcomeScreen.vala:28
 msgid "Add an account"
-msgstr ""
+msgstr "Dodaj konto"
 
 #: src/view/WelcomeScreen.vala:29
 msgid "You have not configured any account yet"
-msgstr ""
+msgstr "Nie skonfigurowałeś jeszcze żadnego konta"
 
 #: src/view/AccountWelcomeScreen.vala:41
 msgid "Spend! Get paid!"
-msgstr ""
+msgstr "Wydawaj! Zarabiaj!"
 
 #: src/view/AccountWelcomeScreen.vala:41
 msgid "There are currently no transactions in this account"
-msgstr ""
+msgstr "Aktualnie braj transakcji dla tego konta"
```

Comment:
Google Translate suggests using  "Aktualnie **brak** transakcji dla tego konta"

Reply:
Yup, this was a typo, sorry.